catalog/app/containers/Admin/RFForm.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+import * as M from '@material-ui/core'
+
+type FieldProps = M.TextFieldProps & {
+  meta: $TSFixMe
+  input: {
+    value: $TSFixMe
+    onChange: (value: $TSFixMe) => void
```

Comment:
I see, that you are in a hurry :)
So, I'm approving now and review later, even after merging

Reply:
the problem with this is more fundamental -- we need a way to define form typings properly, so every field is strongly typed wrt field value, validator, initial value, errors and such (smth like this probably: https://github.com/ForbesLindesay/define-form/tree/master/packages/react-define-form), otherwise it doesnt make much sense